Dockerfile

```diff
@@ -25,30 +25,30 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 # fetch the cBioPortal sources and version control metadata
 ENV PORTAL_HOME=/cbioportal
-RUN git clone --depth 1 -b v1.16.0 'https://github.com/cBioPortal/cbioportal.git' $PORTAL_HOME
+RUN git clone --depth 1 -b v1.16.0+backport4787 'https://github.com/thehyve/cbioportal.git' $PORTAL_HOME
 WORKDIR $PORTAL_HOME
 
 #RUN git fetch --depth 1 https://github.com/thehyve/cbioportal.git my_development_branch \
 #       && git checkout commit_hash_in_branch
 
 # add buildtime configuration
-COPY ./portal.properties src/main/resources/portal.properties
 COPY ./log4j.properties src/main/resources/log4j.properties
 
-# install default config files, build and install, placing the scripts jar back
-# in the target folder where import scripts expect it after cleanup
+# build and install, placing the scripts jar back in the target folder
+# where import scripts expect it after cleanup
 RUN mvn -DskipTests clean package \
 	&& unzip portal/target/cbioportal-*.war -d $CATALINA_HOME/webapps/cbioportal \
 	&& mv scripts/target/scripts-*.jar /root/ \
 	&& mvn clean \
 	&& mkdir scripts/target/ \
 	&& mv /root/scripts-*.jar scripts/target/
 
-# add runtime configuration
+# add runtime plumbing to Tomcat config:
+# - make cBioPortal honour db config in portal.properties
+RUN echo 'CATALINA_OPTS="$CATALINA_OPTS -Ddbconnector=dbcp"' >>$CATALINA_HOME/bin/setenv.sh
```

Comment:
Yes, I definitely hope to steer that way. I've discussed the idea in the community via several channels, and people generally seem to agree that the Tomcat resource is a niche use-case

Even if all existing deployers and automation scripts rely on the current default, it might not be too hard: they'd hopefully already have the right credentials in a `portal.properties` file even if it's just for the data loading scripts.

Documenting the use-case for`-Ddbconnector=dbcp` is being tracked in https://github.com/cBioPortal/cbioportal/issues/4832, and was already being used in Heroku deployments.

Reply:
Ok, we can do this for now. Should I open an issue here so we don't lose track and remember to remove it later?